.github/workflows/backend-testing.yaml

```diff
@@ -0,0 +1,20 @@
+name: Run tests in BackendApp
+run-name: ${{ github.head_ref }} is testing django app 🚀
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+jobs:
+  Run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: pip install -r BackendApp/requirements.txt
+      - name: Test
+        working-directory: BackendApp
+        run: python3 manage.py test --verbosity 2
```

Comment:
newline - have you installed official GitHub Actions VSCode extension? It helps with formatting.